introduction.md

```diff
@@ -6,33 +6,31 @@ layout: documentation
 
 # About openHAB
 
-openHAB is **software** for integrating different home automation systems and technologies into one single solution that allows over-arching automation rules, and that offers uniform user interfaces.
+openHAB is **software** for integrating different home automation systems, devices and technologies into one single solution.
```

Comment:
Maybe
> openHAB is **a** software

Not necessary, but it just sounded better to me during reading. 😄

Reply:
@Confectrician, I wish I had paid more attention during English grammar lessons, but I didn't, so I cant' tell you why, but  putting the "a" in is incorrect.  

That said, you are correct - the sentence is awkward as written.  How about, "openHAB software integrates different...".  This reads more naturally to me.

Reply:
It seems you have at least more attention than me, if you know that its wrong. 😄 
Perfectly fine with your suggestion.

Reply:
This comment has been addressed.